serhii.trokhymets/Home_work 3/task_1.rb

```diff
@@ -0,0 +1,29 @@
+class EasyNumber
+  attr_accessor :prime_arr, :bad_number
+  attr_reader :max_value
+
+  def initialize (max_value)
+    @max_value = max_value
+    @prime_arr = []
+    @bad_number = []
```

Comment:
Why initiate a variable that you do not use? The same is true for the attr_accessor argument